docs/en/api-reference/system/ulp-lp-core.rst

```diff
@@ -161,7 +169,7 @@ To access the ULP LP core program variables from the main program, the generated
 .. note::
 
     Variables declared in the global scope of the LP core program reside in either the ``.bss`` or ``.data`` section of the binary. These sections are initialized when the LP core binary is loaded and executed. Accessing these variables from the main program on the HP-Core before the first LP core run may result in undefined behavior.
-
+    The ``ulp_`` prefix is the default value. You can specify the prefix to use with ``ulp_embed_binary`` to avoid name collusions for multiple ULP program.
```

Comment:
Nitpick:

```suggestion
    The ``ulp_`` prefix is the default value. You can specify the prefix to use with ``ulp_embed_binary`` to avoid name collisions for multiple ULP programs.
```